setup.py

```diff
@@ -37,6 +39,7 @@
 ]
 
 doc_reqs = [
+    "docutils==0.20",
     "m2r2",
```

Comment:
Can't you have a more thorough solution for this problem?
Have you considered using `sphinx-mdinclude` package instead of `m2r2`?